src/screens/Inventory/PurchaseRequest.tsx

```diff
@@ -28,23 +28,25 @@ const styles = (theme: Theme) =>
 interface PurchaseRequestsProps extends RouteComponentProps {
   classes: { content: string; section: string; };
   location: any;
-  product: ProductRecord;
+  // product: ProductRecord;
 }
 
 
 function PurchaseRequest (props: PurchaseRequestsProps) {
-  const { classes, product } = props;
+  const { classes } = props;
   const purchaseRequest: PurchaseRequestRecord = props.location.state.purchaseRequest;
+  const product = useSelector((state: RootState) => selectProductById(state, selectCurrentSiteInventoryById(state, purchaseRequest.inventoryId)?.productId || ""))|| EMPTY_PRODUCT;
```

Comment:
Is there any way this can be split into 2 lines? The line is 180 characters and is kinda hard to read